docs/features/update-line-items-webhook.md

```diff
@@ -33,6 +34,11 @@ curl --location --request POST 'http://simple-roster.docker.localhost/api/v1/web
 }'
 ```
 
+#### Authentication
+
+To use this endpoint, basic authentication should be used based on `WEBHOOK_USERNAME` and `WEBHOOK_USERNAME` configured 
+on environment settings.
```

Comment:
Proposal: `...on environment settings. And providing those credentials as a header in request from delivery side`

Reply:
The text was a little off... Had reworded it a little.